django_wikidata_api/tests/test_fields.py

```diff
@@ -0,0 +1,155 @@
+# coding=utf-8
+""" Unit tests for models.py """
+from rest_framework.fields import (
+    CharField,
+    Field,
+    ReadOnlyField,
+)
+from django.test import TestCase
+
+from django_wikidata_api.fields import (
+    WikidataCharField,
+    WikidataDescriptionField,
+    WikidataField,
+    WikidataLabelField,
+)
+
+
+class WikidataFieldTests(TestCase):
```

Comment:
In this file, I'm adding tests for the new description field class and all it's parent classes